source/troubleshooting.txt

```diff
@@ -89,8 +89,59 @@ to create new projects.
 
 The user's IP address or API key that you use to authenticate is not on
 the :atlas:`access list </configure-api-access>` for the requested
-project. Add your IP address or API key to the
-:atlas:`access list </configure-api-access>` to run commands.
+project. Add your IP address or API key to the :atlas:`access list 
+</configure-api-access>` to run commands.
+
+To learn more, see the following pages:
+
+- For project access lists, see 
+  :atlas:`Configure IP Access List Entries </security/ip-access-list/>`.
+
+- For API key access lists, see 
+  :atlas:`Get Started with the Atlas Administration API 
+  </configure-api-access>`.
+
+To add your IP address to an API key's access list:
+
+.. procedure::
+   :style: normal
+
+   .. step:: Go to the :guilabel:`Access Manager` page for your project.
+
+      a. If it isn't already displayed, select the organization that
+         contains your desired project from the :icon-mms:`office` 
+         :guilabel:`Organizations` menu in the navigation bar.
+
+      #. Select your desired project from the list of projects in the 
+         :guilabel:`Projects` page.
+
+      #. Next to the :guilabel:`Projects` menu, expand the
+         :icon-fa5:`ellipsis-v` :guilabel:`Options` menu, then click 
+         :guilabel:`Project Settings`.
+
+      #. Click :guilabel:`Access Manager` in the navigation bar.
+
+   .. step:: Go to the API key's access list.
+
+      a. Click the :guilabel:`API Keys` tab.
+
+      #. Click :icon-mms:`ellipsis` to the right of the |api| Key.
+  
+      #. Click :guilabel:`Edit Permissions` and click :guilabel:`Next`.
+      
+   .. step:: Add your IP address to the API key's access list.
+
+      a. Do one of the following tasks in the 
+         :guilabel:`API Access List` section:
+
+         - Click :guilabel:`Add Access List Entry` and type an |ipaddr| 
+           address.
+
+         - If your current host for accessing |service| will also make 
+           |api| requests with this |api| key, click 
```

Comment:
I think we don't need this section now that we have the link to the correct section on the other page.

Reply:
I recommend that we leave these exact steps in for clarity and ease of use since there was a lot of back and forth on this.